full-moon/src/tokenizer.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(clippy::unnecessary_wraps)] // Clippy currently triggers false positives wrt. Advancement: https://github.com/rust-lang/rust-clippy/issues/6613
```

Comment:
Where was this popping up? Only in a few places, or everywhere? If it's not in a lot of places, I'd rather filter those places specifically.

Reply:
It was for all the following:
- `advance_identifier`
- `advance_shebang`
- `advance_symbol`
- `advance_whitespace`
The only one it was fine for was `advance_quote`

Reply:
Following the merge of #154, this is no longer necessary. It has now been removed